projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.ts

```diff
@@ -436,7 +436,7 @@ export class IgxGridExcelStyleFilteringComponent implements OnDestroy {
      */
     public onSelect() {
         if (!this.column.selected) {
-            this.grid.selectionService.selectColumn(this.column.field);
+            this.grid.selectionService.selectColumn(this.column.field, this.grid.columnSelection === 'single');
```

Comment:
Use the GridSelectionMode enumeration instead of string value

`this.grid.selectionService.selectColumn(this.column.field, this.grid.columnSelection === GridSelectionMode.single)`